sea_itt_lib/Utils.cpp

```diff
@@ -110,8 +110,11 @@ size_t GetStack(TStack& stack)
     static FCaptureStackBackTrace CaptureStackBackTrace = (FCaptureStackBackTrace)(GetProcAddress(LoadLibraryA("kernel32.dll"), "RtlCaptureStackBackTrace"));
     return CaptureStackBackTrace ? CaptureStackBackTrace(0, StackSize, stack, NULL) : 0;
 #else
-    //return backtrace(stack, StackSize);
-	return 0;
+    #ifdef USE_MUSL_LIBC
+        return 0;
+    #else
+        return backtrace(stack, StackSize);
```

Comment:
Why was this commented out before and now it is not. If USE_MUSL_LIBC is not defined should this code really be executed?

Reply:
Same as above: ; non-MUSL-libc code should be able to run `backtrace(...)`.